### ISSUE TITLE
Documentation for Accounts and Notifications

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/chapter#262]
**Account & Notifications**
- [ ] Accessing an account
- [ ] Resetting your password
- [ ] Organizer subscriptions
- [ ] Notification settings

**Comment by allella** [comment]
@chrismgonzalez 

- If the MVP is using Google for authentication, then is documenting an account page or password reset still applicable? I guess we could document the fact that your email and password are managed by Google, so visit https://myaccount.google.com or https://myaccount.google.com/security "Signing in to other sites" section? 
- The MVP hasn't mentioned an account page, but if we do have one then can we imagine is needed?  #274 and [the suggestions about subscribing and unsubscribing may avoid the need for an account page](https://github.com/freeCodeCamp/chapter/issues/107#issuecomment-562374895) to managed subscriptions, at least for MVP.
- What are we thinking with "Organizer subscriptions"? If it's for the organizer, then we probably need to move that item to #260.  Or, was this intended to be "Organize Subscriptions"?

**Comment by QuincyLarson** [comment]
@allella These are great points. I've updated the bullet points above to reflect this by removing the need for password management.

We should have a profile/settings page, and the ability to make it private. If you can document what specific attributes the page should have, we can ask @madaleneaza-design if she can create a layout for this.

**Comment by allella** [comment]
I would agree that a basic profile management page is necessary IF any of the following are true:
- [we're not pulling a _user's_ name from Google's profile or we use the Google name, but allow the user to override their name](https://github.com/freeCodeCamp/chapter/issues/110#issuecomment-566292771)
- the user needs a public profile page. The alternative is we simply show the _user's_ name on the _RSVP_ list.
- privacy is set at the_user_ profile level.  If privacy is set at the _chapter_-level, then we'd need a privacy profile settings page.  If privacy is set at at the _event_ level only, then we'd don't necessarily need a _user_ level privacy setting.  

I've shared my opinion that a subscription management / settings page is not a necessary MVP feature for a [number or reasons outlined earlier].(https://github.com/freeCodeCamp/chapter/issues/274#issuecomment-565740617).

I've figured out that "Organizer subscriptions" in the checklist above was likely copied from [Meetup's settings page](https://www.meetup.com/account/subscription/). If that's the case then "Organizer Dashboard" and "Admin Dashboard" are more appropriate to #260 since everything is free and organizers don't pay a subscription.

**Comment by allella** [comment]
Alright, to summarize the conversation from today related to the Settings / Account / Notifications page a _user_ will see:

- We'll want to have links to the [ToC and Privacy pages](https://github.com/freeCodeCamp/chapter/issues/263#issuecomment-1212177145) somewhere in the app. Are we going to have a footer, or shall we add links to the account page?
- [Related to GDPR](https://github.com/freeCodeCamp/chapter/issues/263#issuecomment-1212177145), we should have a section on the settings / account page showing what personal information (name, email) has been collected
- For alpha, we'll add a button for Account Deletion requests that emails fCC support for manual deletion until we can create a more automated approach. When an account is deleted, we'll likely show something like "Deleted Member" on event pages where deleting the record would otherwise distort attendance counts and such
- Rather than having a separate document explaining the user's controls and settings, we'll use a [GitHub settings](https://github.com/settings/profile) and [fCC settings](https://www.freecodecamp.org/settings) style of "self-documenting" pages (example below)


**Example of Github's settings page**

![image](https://user-images.githubusercontent.com/1777776/184179713-c576a443-3d8f-465c-89dc-8d1bc79d9860.png)


**Example of freeCodeCamp's settings page**

![image](https://user-images.githubusercontent.com/1777776/184179845-51bcef4b-23e0-4fd8-95b6-aac7f10382e3.png)
